common/recordings.md

```diff
@@ -100,6 +100,17 @@ You can specify a cloud storage location for your recording files in your templa
 
 The above message ensures that your configuration is successful now, and all your recordings will start collecting in your configured destination.
 
+
+## Google and Alibaba Cloud Storage for Recordings
+
+Google Cloud and Alibaba OSS storage is currently supported through [Rooms](https://www.100ms.live/docs/server-side/v2/api-reference/Rooms/create-via-api) and [Policy](https://www.100ms.live/docs/server-side/v2/api-reference/policy/create-template-via-api) APIs. You can configure the `type` field of recording object to `oss` for Alibaba Object Storage Service and `gs` for Google Cloud Storage with the following details:
```

Comment:
let's only suggest policy APIs. rooms APIs are a not way to do this because creds don't show up on the template.

i think it'll be useful to have an intro section on top where we say
* Supported cloud storage
  * AWS: configure through API or UI
  * GCS etc.: configure through API only for now